cli/exp_task_status_test.go

```diff
@@ -0,0 +1,146 @@
+package cli_test
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"sync/atomic"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/coder/coder/v2/cli/clitest"
+	"github.com/coder/coder/v2/coderd/httpapi"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/coder/v2/testutil"
+)
+
+func Test_TaskStatus(t *testing.T) {
+	t.Parallel()
+
+	for _, tc := range []struct {
+		args         []string
+		expectOutput string
+		expectError  string
+		hf           func(context.Context) func(http.ResponseWriter, *http.Request)
```

Comment:
review: I elected to mock the API response instead of standing up a whole `coderdtest`. We have too many of these as it is.